sdk/python/setup.py

```diff
@@ -67,4 +69,6 @@
     ],
     python_requires='>=3.5.3',
     include_package_data=True,
-    entry_points={'console_scripts': ['dsl-compile = kfp.compiler.main:main',]})
+    entry_points={'console_scripts': [
+        'dsl-compile = kfp.compiler.main:main',
```

Comment:
What do you think of exposing one command line tool for both compilation and client submission?
for example: kfp compile(=dsl-compile). kfp client(=kfp)

Reply:
I am going to add "kfp compile" as the next PR and "kfp run" can also support to start a run directly from python file.

Reply:
Perfect. 
Does that mean the dsl-compile tool will be deprecated, then?

Reply:
I think so. I will add deprecation warning in the dsl-compile after adding kfp compile.

Reply:
SG